governance/votes/Executive vote - November 29, 2023.md

```diff
@@ -0,0 +1,146 @@
+
+
+### DAI transfer to cover Andromeda Legal expenses
+
+*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/project-andromeda-legal-expenses-ii/22577/4)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/project-andromeda-legal-expenses-ii/22577) "Project Andromeda - Legal Expenses II"
+
+If this executive proposal passes, the following transfer will be performed to cover Andromeda (RWA015-A) Legal expenses:
+
+- *201,738 DAI* will be transfered to [0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767](https://etherscan.io/address/0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767).
+
+
+### Cease MKR and DAI Vest Streams for Steakhouse and TechOps
+
+*Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)\
+
+If this executive proposal passes, in line with the recent amendments in the [Governance Scope](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a), the following DSS Vest Streams will be halted by calling the Yank function:
+
+- Steakhouse *DAI* stream *ID 19*
+- TechOps *DAI* stream *ID 18*
+- Steakhouse *MKR* stream *ID 32*
+- TechOps *MKR* stream *ID 33*
+
+
+### Set up MKR and DAI Vest Streams for Ecosystem Facilitator
+
+*Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)\
+
+If this executive proposal passes, in line with the recent amendments in the [Governance Scope](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a), the following DSS Vest Streams will be set up for Ecosystem facilitator:
+
+| Recipient | Stream Dates                                | Amount     | Token | Address                                                                                                               |
+|-----------|---------------------------------------------|------------|-------|-----------------------------------------------------------------------------------------------------------------------|
+| Ecosystem | 2023-12-01 00:00:00 to 2024-11-30 23:59:59 | 504,000.00 | DAI   | [0xFCa6e196c2ad557E64D9397e283C2AFe57344b75](https://etherscan.io/address/0xFCa6e196c2ad557E64D9397e283C2AFe57344b75) |
+| Ecosystem | 2023-12-01 00:00:00 to 2024-11-30 23:59:59 |     216.00 | MKR   | [0xFCa6e196c2ad557E64D9397e283C2AFe57344b75](https://etherscan.io/address/0xFCa6e196c2ad557E64D9397e283C2AFe57344b75) |
+
+
+### Spark Protocol Proxy-Spell
+
+*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/accounting-discrepancy-in-the-dai-market/22845/2)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/accounting-discrepancy-in-the-dai-market/22845) "Accounting Discrepancy in the DAI Market"
+
+If this executive proposal passes, a Spark Protocol proxy-spell with address [0x68a075249fA77173b8d1B92750c9920423997e2B](https://etherscan.io/address/0x68a075249fA77173b8d1B92750c9920423997e2B) will be excecuted to address a identified discrepancy in the DAI Market.
+There is no impact of this issue on users and its severity is classified as low.
```

Comment:
```suggestion
If this executive proposal passes, a Spark Protocol proxy-spell with address [0x68a075249fA77173b8d1B92750c9920423997e2B](https://etherscan.io/address/0x68a075249fA77173b8d1B92750c9920423997e2B) will be excecuted to address a identified discrepancy in the DAI Market.

There is no impact of this issue on users and its severity is classified as low.
```

If a line break is intended here then you need an empty line in markdown, or to use a double space "  " at the end of the first line.